opacclient/opacapp/src/main/res/layout/content_accountitem_detail.xml

```diff
@@ -90,7 +90,7 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:padding="16dp"
-                    android:visibility="@{item.author != null ? View.VISIBLE : View.GONE}">
+                    android:visibility="@{(item.author != null &amp;&amp; !item.author.equals(&quot;&quot;)) ? View.VISIBLE : View.GONE}">
```

Comment:
Shouldn't we ensure that `author == null || !author.equals("")` in the APIs (e.g. in the Unit Tests, similarly for all the other properties)? Always checking for both `null` and empty Strings makes code like this somewhat ugly :/